tools/bots/mission-control-bot/actions.ts

```diff
@@ -147,6 +147,7 @@ export async function botActionBalance(web3Api: Web3, msg: Message, messageConte
 export async function balanceMonitorCheck(web3Api: Web3, workers: WorkerAccount[]) {
   const alertThreshold = params.BALANCE_ALERT_THRESHOLD * 10n ** TOKEN_DECIMAL;
   const workerMinBalance = params.WORKER_MIN_BALANCE * 10n ** TOKEN_DECIMAL;
+  const overBalance = 5n * params.TOKEN_COUNT * 10n ** TOKEN_DECIMAL;
```

Comment:
What is this one for ?

Reply:
Users requesting tokens are funded by worker accounts, which are funded in a regular basis from a master funds account. Here is where the workers are funded if their balance goes below a specific threshold. 

Before we were funding the workers with the difference between the current balance and the threshold, which was fine, but required us to fund it again right after just 1 fund transaction from the worker to a user. 

By incrementing the amount sent to the worker when it goes below the threshold (by `overBalance`, which is set at 5 times the amount usually sent on one fund transaction), we lower the amount of transactions sent from the master funds account, resulting in less gas wasted.